earth_enterprise/src/common/khxml/khxml.cpp

```diff
@@ -40,50 +41,147 @@ ListElementTagName(const std::string &tagname)
 // This is used only in the following function
 class UsingXMLGuard
 {
-  friend void InitializeXMLLibrary(void) throw();
-
-  UsingXMLGuard(void) throw() {
-    try {
-      XMLPlatformUtils::Initialize();
+  friend void InitializeXMLLibrary(bool) throw();
```

Comment:
I can see the value in keeping this code handy for further testing, but I don't think we want it in the master branch. Can we move this off to a separate WIP PR or something like that?

Reply:
The same goes for all of the changes to initializing the XML library.

Reply:
Sure thing